CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.7.0 (2021-04-01)
```

Comment:
```suggestion
# 4.7.0 (2021-04-02)
```
Assuming we won't actually merge this until tomorrow. If you catch it tonight, disregard this edit.